rmgpy/solver/base.pyx

```diff
@@ -621,6 +625,7 @@ cdef class ReactionSystem(DASx):
         sensitivityRelativeTolerance = simulatorSettings.sens_rtol
         filterReactions = modelSettings.filterReactions
         maxNumObjsPerIter = modelSettings.maxNumObjsPerIter
+        trimolecular = False if self.trimolecularThreshold is None else True
```

Comment:
I'm a bit confused.  In general structurally are you trying to infer that trimolecular reactions aren't happening because of the value of self.trimolecularThreshold or is trimolecular itself the input that determines this?  If it's the second I think you should create and check self.trimolecular instead of checking if self.trimolecularThreshold is None.